Objective-C/CBLReplicatorConfiguration.m

```diff
@@ -89,8 +99,7 @@ - (instancetype) initWithTarget:(id<CBLEndpoint>)target {
     self = [super init];
     if (self) {
         _target = target;
-        
-        // TODO: Add implementation
+        _collectionConfigs = [NSMutableDictionary dictionary];
```

Comment:
Need to initialize the same properties as initWithDatabase:target does. Try to extract them out in a method.